packages/host/app/components/ai-assistant/apply-button/usage.gts

```diff
@@ -23,6 +23,9 @@ export default class AiAssistantApplyButtonUsage extends Component {
         this.state = 'failed';
         break;
       case 'failed':
+        this.state = 'preparing';
+        break;
+      case 'preparing':
```

Comment:
Freestyle-only, but I noticed that clicking to cycle through these doesn’t actually work because some of the elements aren’t reactive:

![screencast 2025-03-20 09-18-23](https://github.com/user-attachments/assets/b8c2e54d-ada4-4dd7-9078-f2a5e601773a)